independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/commands/handlers/CreateProjectCommandHandler.java

```diff
@@ -75,6 +90,36 @@ public QuarkusCommandOutcome execute(QuarkusCommandInvocation invocation) throws
         List<Extension> extensionsToAdd = computeRequiredExtensions(invocation.getExtensionsCatalog(), extensionsQuery,
                 invocation.log());
 
+        // Resolves the parent project if it exists. Working for maven projects only
```

Comment:
I know this is sort of a point in time thing.. but I don't like this big blob of maven specific code in the middle of a generic command handler. 

I have to look again, but I might suggest a different location for this so it can be validated earlier/differently by the CLI.

Reply:
Hey I haven't seen the code for a while. It felt quite dodgy for me by then. If you think there is a better place to add the code I will change it.

Reply:
https://github.com/quarkusio/quarkus/blob/main/independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/project/codegen/CreateProjectHelper.java -- that is a generic place (note static helper methods). 

These methods are used (more granularly) by the CLI during parameter validation: https://github.com/quarkusio/quarkus/blob/main/devtools/cli/src/main/java/io/quarkus/cli/create/CodeGenerationGroup.java

Reply:
@ebullient I can not see how much value can wee add by moving this to the helper class.

I mean I can do it and kind of understand the reason, but at the end, as you said it will look like a hack, just in a different place. There is not an strategy in place for BuiltTool specifics operations and this will make me to move the logic to https://github.com/quarkusio/quarkus/blob/ed22adc9705ca89ef675b13130907470e778ff49/independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/commands/CreateProject.java#L207 
if the intention is to remove any responsibility on CreateProjectCommandHandler for such operations. 

@ia3andy do you have any thoughts on this ?

Reply:
I think it's fine for now, maybe @ebullient and I could give a clean pass on those in another PR?

Reply:
@cristian-com could you move this block to a dedicated method please?

Reply:
@ia3andy sorry I did this yesterday, could you please check ?